admin-default.properties

```diff
@@ -71,6 +71,7 @@ mosip.admin.security.policy.userrole-auth-url=${mosip.kernel.authmanager.url}/v1
 
 ## Masterdata cards
 
+mosip.optional-languages=fra,ara,spa,tam,hin,kan,es
```

Comment:
This should be part of application-default.properties. As this is common property used by most of the services in mosip.

Reply:
I want this property here temporarily for testing one bug reported by dissemination team anusha, I added this property here such that it won't impact any other service.

Reply:
updated the same as suggested.